public/index.html

```diff
@@ -3,9 +3,10 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#000000">
+    <meta name="theme-color" content="#55B8BE">
```

Comment:
to better align with navbar color of webui: https://github.com/ipfs-shipyard/ipfs-css/blob/main/README.md#colors

```suggestion
    <meta name="theme-color" content="#0b3a53">
```